app/templates/events/event_list.html

```diff
@@ -69,22 +69,22 @@ <h1 class="text-center">Events List for {{selectedTerm.description}}</h1>
     </thead>
     <tbody>
         {% for event in events %}
-          <tr>
+        {% if not event.isPast %}
             {% if type != "bonner" %}
               {% if event.singleProgram%}
-                <td>{{event.singleProgram.programName}}</td>
+                <td  class=text-primary>{{event.singleProgram.programName}}</td>
               {% else %}
-                <td>This event does not belong to any program.</td>
+                <td  class=text-primary >This event does not belong to any program.</td>
               {% endif %}
             {% endif %}
             {% if user.isCeltsAdmin or user.isProgramManagerFor(event.singleProgram)%}
-              <td> <a href="/eventsList/{{event.id}}/edit" class="link-primary">{{event.name}}</a></td>
+              <td> <a href="/eventsList/{{event.id}}/edit" class=text-primary>{{event.name}}</a></td>
             {% else %}
-              <td> <a href="/eventsList/{{event.id}}/view" class="link-primary">{{event.name}}</a></td>
+              <td> <a href="/eventsList/{{event.id}}/view" class=text-primary>{{event.name}}</a></td>
             {% endif %}
-            <td nowrap>{{event.startDate.strftime('%m/%d/%Y')}}</td>
-            <td nowrap>{{event.timeStart.strftime('%-I:%M %p')}} - {{event.timeEnd.strftime('%-I:%M %p')}}</td>
-            <td>{{event.location}}</td>
+            <td nowrap  class=text-primary >{{event.startDate.strftime('%m/%d/%Y')}}</td>
```

Comment:
You can't have text be blue unless it is clickable. In this case it is event more confusing since the event name is a link.